device/celestica/x86_64-cel_belgite-r0/installer.conf

```diff
@@ -0,0 +1,4 @@
+CONSOLE_PORT=0x3f8
+CONSOLE_DEV=0
+CONSOLE_SPEED=115200
```

Comment:
@lguohan do we need `9600` for better compatibility?

Reply:
Hi Blueve, seems our Banyan usually use the 9600 Baud Rate, also, I will update it later. Thanks Again.